tests/pcap/src/lib.rs

```diff
@@ -5,13 +5,25 @@ pub mod client_hello;
 pub mod handshake_message;
 pub mod packet;
 
-/// Creates an iterator over all test pcap file paths
-pub fn all_pcaps() -> impl Iterator<Item = String> {
-    std::fs::read_dir("data")
-        .expect("Missing test pcap file")
+use std::path::Path;
+
+fn pcaps_from<P: AsRef<Path>>(path: P) -> impl Iterator<Item = String> {
+    std::fs::read_dir(path.as_ref())
+        .unwrap_or_else(|_| panic!("Unable to read pcap folder: {}", path.as_ref().display()))
         .filter_map(Result::ok)
         .filter_map(|entry| {
             let path = entry.path();
             path.to_str().map(std::string::ToString::to_string)
         })
 }
+
+/// Creates an iterator over all test pcap file paths
+pub fn all_pcaps() -> impl Iterator<Item = String> {
+    let local_pcaps = pcaps_from("data");
+
+    let downloaded_pcaps_path =
+        std::env::var("DOWNLOADED_PCAPS_PATH").expect("DOWNLOADED_PCAPS_PATH not set by build.rs");
```

Comment:
It would be better to use the [`env!` macro](https://doc.rust-lang.org/std/macro.env.html), since `std::env::var` is a runtime value. I'm actually surprised this is working TBH, since I didn't think `build.rs` environment variables get set when the test actually runs.

Reply:
It does work, although on a second read of the docs it's marked discouraged (:

https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-env
> Note: These environment variables are also set when running an executable with cargo run or cargo test. However, this usage is discouraged since it ties the executable to Cargo’s execution environment. Normally, these environment variables should only be checked at compile-time with the env! macro.

But I also added a test so that there's no doubt that it's working.